presto-main/src/main/java/com/facebook/presto/operator/index/IndexBuildDriverFactoryProvider.java

```diff
@@ -39,6 +39,7 @@
     private final List<OperatorFactory> coreOperatorFactories;
     private final List<Type> outputTypes;
     private final Optional<DynamicTupleFilterFactory> dynamicTupleFilterFactory;
+    private static final String INDEX_BUILDER = "IndexBuilder";
```

Comment:
Move this to the top of the stack. Break a line between static and non-static members.